apis/config/v1beta1/configuration_types.go

```diff
@@ -169,6 +169,17 @@ type ControllerMetrics struct {
 	// metrics will be reported.
 	// +optional
 	EnableClusterQueueResources bool `json:"enableClusterQueueResources,omitempty"`
+
+	// +optional
+	LocalQueueMetrics *LocalQueueMetrics `json:"localQueueMetrics,omitempty"`
+}
+
+type LocalQueueMetrics struct {
```

Comment:
What would be the cost and implication of having it enabled for everyone and every LQ?

Reply:
Total number of active Prometheus time series would be `# of namespaces` x `# of unique lq names` x `# resources` x `# of flavors`

Which can overwhelm prometheus metrics server depending on the amount of RAM available